sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -848,6 +852,45 @@ def execute_query_raw(self, query):
             cursor.execute(query)
             return cursor.fetchall()
 
+    # https://learn.microsoft.com/en-us/sql/relational-databases/system-dynamic-management-views/sys-dm-db-index-usage-stats-transact-sql?view=sql-server-ver15
+    def collect_index_usage_metrics(self):
+        if not is_affirmative(self.instance.get('include_index_usage_metrics', True)):
+            return False
```

Comment:
nit: returning a value is unnecessary and somewhat confusing. Can just `return` and `return None` is implied.